SpatialGDK/Documentation/content/handover-between-server-workers.md

```diff
@@ -32,7 +32,7 @@ To facilitate an Actor’s property handover between server-workers, follow the
 
 1. Generate the [schema]({{urlRoot}}/content/glossary#schema-generation) for your Actor’s class. (In the Unreal Editor, from the [GDK toolbar]({{urlRoot}}/content/toolbars), select the **Schema** icon.)
 
-The GDK now ensures that server-workers transfer these tagged Actor’s properties between them.
+The GDK now ensures that server-workers transfer these tagged Actor’s properties between them. You can also define custom behaviour which executes when authority on such properties is lost or gained via [authority callbacks]({{urlRoot}}/content/authority#authority-callbacks).
```

Comment:
This isn't quite accurate *yet*, we transfer handover properties continuously, but only issue the authority callback when authority changes.

Reply:
Also we don't talk about authority over properties in any other parts of the documentation. Authority is usually in the context of over an Actor or over the ClientRPC component.